babel/src/board.js

```diff
@@ -247,14 +244,14 @@ export class Board {
           * 3. if the row isn't 0, it takes the orb from above
           * 4. if the row is 0, it creates a random orb
         */
-        _.each(_.range(this.height), x =>{
-            _.each(_.range(this.width), y => { //1
-                if (this.orbs[x][y] == 'X') {
-                    for (var z = x; z >= 0; z--) { //2
+        _.each(_.range(this.height), row =>{
+            _.each(_.range(this.width), col => { //1
+                if (this.orbs[row][col] == 'X') {
```

Comment:
I'll raise an issue later -- you shouldn't use a ["magic string"](https://en.wikipedia.org/wiki/Magic_string) like `"X"` here, it can get nasty if I were to specify my orb types as `['W', 'X', 'Y', 'Z']`!